src/screens/UserPortal/Pledges/Pledges.tsx

```
@@ -214,15 +214,15 @@ const Pledges = (): JSX.Element => {
                   ) : (
                     <div className={styles.avatarContainer}>
                       <Avatar
-                        key={user._id + '1'}
+                        key={user.id + '1'}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)